hack/generated/controllers/generic_controller.go

```diff
@@ -562,36 +578,37 @@ func (gr *GenericReconciler) constructArmResource(ctx context.Context, data *Rec
 	return resource, nil
 }
 
-func (gr *GenericReconciler) getStatus(ctx context.Context, id string, data *ReconcileMetadata) (genruntime.FromArmConverter, error) {
+func (gr *GenericReconciler) getStatus(ctx context.Context, id string, data *ReconcileMetadata) (genruntime.FromArmConverter, error, time.Duration) {
 	deployableSpec, err := reflecthelpers.ConvertResourceToDeployableResource(ctx, gr.ResourceResolver, data.metaObj)
 	if err != nil {
-		return nil, err
+		return nil, err, 0
```

Comment:
minor: Might be clearer to use `time.Duration(0)` here? Not sure if that's idiomatic or not though

Reply:
Same thoughts below